gourmet/timeScanner.py

```diff
@@ -67,7 +66,6 @@ def show_timer_cb (tv,l,note,c):
                note)
 
 if __name__ == '__main__':
-
     from gi.repository import Gtk
```

Comment:
Delete this one and leave it at the top of the file?

Reply:
Will do. I deleted the top Gtk import as it is only used in `__main__` and not in timeScanner module per se.